imports/ui/dialogs/create-report-dialog.jsx

```diff
@@ -25,12 +26,26 @@ class CreateReportDialog extends Component {
       }
     }, 300)
   }
+
+  creationTime () {
+    const creationTime = (new Date()).toISOString()
+    return moment(creationTime).format('DD/MM/YYYY')
+  }
+
+  populateTitle = () => {
+    const { reportTitle } = this.state
+    if (reportTitle === '') {
+      this.setState({reportTitle: `${this.creationTime()} ${this.props.unitName}`})
```

Comment:
The title string repeats itself in multiple places. The string should be captured by a class member/const and not just the `creationTime`